scripts/play.ts

```diff
@@ -0,0 +1,380 @@
+import mri from "mri";
+import chalk from "chalk";
+import * as fs from "node:fs/promises";
+import * as fsSync from "node:fs";
+import * as path from "node:path";
+import * as crypto from "node:crypto";
+
+import { exec } from "./utils/commandUtils";
+
+const PLAYGROUNDS_ROOT = path.join(process.cwd(), "playgrounds");
+const DEFAULT_FRAMEWORK = "next" satisfies Args["framework"];
+
+run();
+
+type Args = {
+  "dry-run": boolean;
+  start: boolean;
+  new: boolean;
+  framework?: "next" | "nuxt" | "sveltekit" | (string & {});
```

Comment:
It means "any string." The `{}` is kind of [a TypeScript hack](https://github.com/Microsoft/TypeScript/issues/29729#issuecomment-567871939) to ensure `string` doesn't clobber the other string literal union type.

```ts
type WithoutHack = "next" | "nuxt" | "sveltekit" | string
// => string

type WithHack = "next" | "nuxt" | "sveltekit" | (string & {})
// => "next" or "nuxt" or "sveltekit" or any string
```

We need the "or any string" part of this type since users can pass in any string. Literal types are still useful for autocomplete, however.